### ISSUE TITLE
[Lab 2] índice ppda

### ISSUE THREAD

**Issue by cristobalgon** [IIC2115/Syllabus#8]
Hola!

No entiendo bien qué significa el índice ppda. Se refiere a que mientras más bajo, el equipo que estamos observando tiene una mayor presión defensiva? en ese caso, estaríamos diciendo que el equipo contrincante hace mucho daño?

O es al revés, que mientras menor es el índice, manos es la presión que se ejerce en mi defensa y así el equipo puede ir a atacar más seguido?

Muchas gracias por el tiempo!

**Comment by joaquineroldan** [comment]
Yo lo vi como la presión que ejerce el equipo hacia el contrincante, al ser menor era que el equipo contrario dio menos pases, asi que es mejor defensa

**Comment by halobel** [comment]
Exacto, el ppda se calcula como la razón entre la cantidad de pases del adversario en campo del equipo observado y las acciones defensivas realizadas por el equipo observado. Por lo tanto, si el valor es bajo, significa que la defensa "hizo harto" por cada pase del contrincante (crece el denominador), lo que es un proxy para la presión defensiva.

**Comment by cristobalgon** [comment]
Perfecto, me quedo claro! Gracias!
